docs/source/projects/architecture/content-architecture.rst

```diff
@@ -14,8 +23,8 @@ This section focuses on IPython and kernels.
 When we discuss ``IPython``, we talk about two fundamental roles:
 
 - Terminal IPython as the familiar REPL
-- The IPython kernel that provides computation and communication with the
-  frontend interfaces, like the notebook
+- The IPython kernel, ``IPykernel`` that provides computation and communication with
```

Comment:
Non-blocking note here: this is why I like using "one line per sentence". Avoids arbitrary re-line-breaking changes like this :-)